spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:email) { |n| "test-#{n}@example.com" }
+    password 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.